src/components/form/Form.tsx

```diff
@@ -114,18 +114,30 @@ export function FormPage({ currentPageId }: { currentPageId: string | undefined
             id={id}
           />
         ))}
-        <Flex
-          item={true}
-          size={{ xs: 12 }}
-          aria-live='polite'
-          className={classes.errorReport}
-        >
-          <ErrorReport
-            renderIds={errorReportIds}
-            formErrors={formErrors}
-            taskErrors={taskErrors}
-          />
-        </Flex>
+        {formErrors.length > 0 || taskErrors.length > 0 ? (
+          <Flex
+            item={true}
+            size={{ xs: 12 }}
+            aria-live='polite'
+            className={classes.errorReport}
+          >
```

Comment:
If you conditionally render the element with `aria-live`, will this still announce errors in screen readers?

Context:
- https://github.com/Altinn/app-frontend-react/issues/903
- https://github.com/Altinn/app-frontend-react/pull/963

Reply:
Whoops! I got around this by always rendering this element (like we did before) instead. :pray: